src/pages/home/components/Sidebar/components/SocialMediaForm/SocialMediaForm.components.tsx

```diff
@@ -0,0 +1,43 @@
+import { ReactNode } from 'react';
+
+import Button from '~components/Button/Button.tsx';
+
+import scss from './SocialMediaForm.module.scss';
+
+import { SocialMediaFormProps } from './SocialMediaForm.types';
+
+export function ConnectAccountButton({
+  onHandleToggleModal,
```

Comment:
Essa `prop` poderia se chamar onOpenModal ou coisa parecida, `handle` é uma plavra usada pra quem lida com o evento, não quem chama ele.

Porem vale dizer, se fosse apenas `onClick`

Reply:
Procurando algumas referencias, tambem, de como renomear, melhores praticas e tals encontrei esses posts falando sobre:

- https://spencerpauly.com/tech/react-naming-conventions-best-practices/
- https://www.freecodecamp.org/news/best-practices-for-react/

Porem para nosso caso por mais que seja uma fn que lida com boolean usar is... ou has... nao faz muito sentido, o que voce indicou eh mais apropriado.

Arrumado!